Database/Seeders/json/roles.json

```diff
@@ -10,7 +10,12 @@
     },
     {
         "name": "Registered",
+        "type": "frontend",
+        "details": "Users who have registered roles can access only public website."
+    },
+    {
+        "name": "test",
```

Comment:
@wri-vikram-wri231 we don't need this one

Reply:
@wri-vikram-wri231 add types as `backend` to other roles which are for `backend`

Reply:
@wri-vikram-wri231 this type should be available while editing the roles.